deploy/examples/create-external-cluster-resources.py

```diff
@@ -652,6 +644,21 @@ def __init__(self, arg_list=None):
             self.cluster = rados.Rados()
             self.cluster.conf_read_file()
         self.cluster.connect()
+        if not self._arg_parser.rgw_pool_prefix and not self._arg_parser.upgrade:
+            self._arg_parser.rgw_pool_prefix = self.DEFAULT_RGW_POOL_PREFIX
+        # when user has not provided rgw pool name during upgrade, get the rgw pool name used during connection
+        if not self._arg_parser.rgw_pool_prefix and self._arg_parser.upgrade:
+            _, val = self.check_user_exist("client.healthchecker")
+            # To get value 'default' which is rgw pool name from 'allow rwx pool=default.rgw.meta'
+            pattern = r"pool=(.*?)\.rgw\.meta"
+            match = re.search(pattern, val)
+            if match:
+                self._arg_parser.rgw_pool_prefix = match.group(1)
+                print(self._arg_parser.rgw_pool_prefix)
+            else:
+                raise ExecutionFailureException(
+                    "failed to get rgw pool name for upgrade"
+                )
```

Comment:
```suggestion
        if not self._arg_parser.rgw_pool_prefix and not self._arg_parser.upgrade:
            self._arg_parser.rgw_pool_prefix = self.DEFAULT_RGW_POOL_PREFIX
        # when user has not provided rgw pool name during upgrade, get the rgw pool name used during connection
        if not self._arg_parser.rgw_pool_prefix and self._arg_parser.upgrade:
            self.run_as_user = self.EXTERNAL_USER_NAME
            _, val = self.check_user_exist("client.healthchecker")
            # To get value 'default' which is rgw pool name from 'allow rwx pool=default.rgw.meta'
            pattern = r"pool=(.*?)\.rgw\.meta"
            match = re.search(pattern, val)
            if match:
                self._arg_parser.rgw_pool_prefix = match.group(1)
                print(self._arg_parser.rgw_pool_prefix)
            else:
                raise ExecutionFailureException(
                    "failed to get rgw pool name for upgrade"
                )
```

The above change is used to get rgq pool name used during connecting the clusters the first time. The above changes avoid any doc changes and make upgrade steps same. Otherwise, we had to ask user to pass the rgw pool name they used during cluster connection.